stable/yugaware/values.yaml

```
@@ -41,7 +41,10 @@ yugaware:
   storageClass: ""
   storageAnnotations: {}
   multiTenant: false
-  serviceAccount: yugaware
+  # Providing service account externally won't create new SA.
+  # It just attaches the clusterrole to it. 
+  # Helpful in Yugabyte Platform GKE App. 
```

```suggestion
  ## Name of existing ServiceAccount. When provided, the chart won't create a ServiceAccount.
  ## It will attach the required RBAC roles to it. 
  ## Helpful in Yugabyte Platform GKE App. 
```

The usual convention is to use two hashes before documentation comments in the values.yaml file.